app/graphql/types/fees/amount_details/volume_range.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Types
+  module Fees
+    module AmountDetails
+      class VolumeRange < Types::BaseObject
+        graphql_name 'FeeAmountDetailsVolumeRange'
+
+        field :per_unit_amount, String, null: false
+        field :flat_unit_amount, String, null: false
```

Comment:
GraphQL/OrderedFields: Fields should be sorted in an alphabetical order within their section. Field `flat_unit_amount` should appear before `per_unit_amount`.